blog/convert-raw-trace-data/index.qmd

```diff
@@ -0,0 +1,223 @@
+| 23                | BUYER_CMSN_AMT     | 0            |   | 23                  | Buyer.Commission           | 0,00         |
+| 24                | BUYER_CPCTY_CD     |              |   | 24                  | Buyer.Capacity             | A            |
+| 25                | SLLR_CMSN_AMT      | 0            |   | 25                  | Seller.Commission          | 0,00         |
+| 26                | SLLR_CPCTY_CD      | A            |   | 26                  | Seller.Capacity            |              |
+| 27                | CNTRA_PARTY_ID     | C            |   | 27                  | Reporting.Party.Type       | D            |
+| 28                | CNTRA_PARTY_GVP_ID |              |   | 28                  | Contra.Party.Indicator     | D            |
+| 29                | LCKD_IN_FL         |              |   | 29                  | Locked.In.Indicator        |              |
+| 30                | ATS_FL             |              |   | 30                  | ATS.Indicator              |              |
+| 31                | SPCL_PR_FL         |              |   | 31                  | Special.Price.Indicator    |              |
+| 32                | TRDG_MKT_CD        | S1           |   | 32                  | Trading.Market.Indicator   | S1           |
+| 33                | PBLSH_FL           | Y            |   | 33                  | Dissemination.Flag         | N            |
+| 34                | SYSTM_CNTRL_DT     | 20130102     |   |                     |                            |              |
+| 35                | SYSTM_CNTRL_NB     | 2000000003   |   |                     |                            |              |
+| 36                | PREV_TRD_CNTRL_DT  |              |   | 34                  | Prior.Trade.Report.Date    | NA           |
+| 37                | PREV_TRD_CNTRL_NB  |              |   | 35                  | Prior.Reference.Number     | NA           |
+| 38                | FIRST_TRD_CNTRL_DT | 20130102     |   | 36                  | First.Trade.Control.Date   | 20220401     |
+| 39                | FIRST_TRD_CNTRL_NB | 2000000003   |   | 37                  | First.Trade.Control.Number | 2            |
+
+: Comparison of table structure {#tbl-comparison}
+
+## Structure of the local SQLite database
+
+In order to work with a merged set of final data, we propose an unified schema for the data, which is oriented on the academic data. The resulting SQLite represents the data in a standardized and normalized form that can be used for further analysis with simple SQL queries in a variety of applications and programming languages. The resulting database shares template and labeling with WRDS and can therefore seamlessly interface with existing program codes for cleaning the data, e.g., as given in Tidy Finance in R (https://gist.github.com/patrick-weiss/3a05b3ab281563b2e94858451c2eb3a4). 
+
```

Comment:
Tidy Finance with R

Reply:
as mentioned above, change to Tidy Finance with R